### ISSUE TITLE
Does forge tell a server you're using impact

### ISSUE THREAD

**Issue by SplatTab** [ImpactDevelopment/ImpactIssues#3320]
I was using the murder module on hypixel and was banned how could they tell its just a render hack they could only guess I was cheating the only other hack I had on was sprint with multi direction off etc

**Comment by Shigetorum635** [comment]
Not really, it's just that your exploits were quite obvious to the anti-cheat.

Hypixel has some tools to stop hacked clients.

**Comment by CesiumCs** [comment]
it absolutely does: https://wiki.vg/Minecraft_Forge_Handshake#ModList, and you got what was coming for cheating

**Comment by SplatTab** [comment]
> it absolutely does: https://wiki.vg/Minecraft_Forge_Handshake#ModList, and you got what was coming for cheating edit to add that impact doesn't actually appear when you look at the mod list inside the client, so forge might not actually send it to the server. still deserved though, and you were probably just being grossly blatant with your render mods

but impact isn't a cheat client its a utility client so how could I be cheating?

**Comment by CesiumCs** [comment]
by misusing it on servers that dont allow it
